db/schema.rb

```diff
@@ -30,20 +41,12 @@
     t.datetime 'remember_created_at'
     t.datetime 'created_at', null: false
     t.datetime 'updated_at', null: false
+    t.integer 'sign_in_count', default: 0, null: false
+    t.datetime 'current_sign_in_at'
+    t.datetime 'last_sign_in_at'
+    t.string 'current_sign_in_ip'
+    t.string 'last_sign_in_ip'
     t.index ['email'], name: 'index_users_on_email', unique: true
     t.index ['reset_password_token'], name: 'index_users_on_reset_password_token', unique: true
```

Comment:
do we allow password reset when we do HPI login? Should this be removed?

Reply:
Probably, but this is not the scope of this PR. This PR just brings the overview pages, there should be a seperate PR focussed on cleaning stuff up